bakerydemo/locations/models.py

```diff
@@ -20,20 +20,20 @@ class OperatingHours(models.Model):
     """
     MONDAY = 'Mon'
```

Comment:
Not sure I follow - weren't we going to delete this duplicate set (and tweak whatever code needed to support the de-duplication?)